include/stream_buffer.h

```diff
@@ -137,7 +143,15 @@ typedef struct StreamBufferDef_t * StreamBufferHandle_t;
  * \defgroup xStreamBufferCreate xStreamBufferCreate
  * \ingroup StreamBufferManagement
  */
-#define xStreamBufferCreate( xBufferSizeBytes, xTriggerLevelBytes )    xStreamBufferGenericCreate( xBufferSizeBytes, xTriggerLevelBytes, pdFALSE )
+
+#if ( configUSE_SB_COMPLETED_CALLBACK == 1 )
+    #define xStreamBufferCreate( xBufferSizeBytes, xTriggerLevelBytes ) \
+        xStreamBufferGenericCreate( xBufferSizeBytes, xTriggerLevelBytes, pdFALSE, NULL)
+    #define xStreamBufferCreate( xBufferSizeBytes, xTriggerLevelBytes, pxStreamBufferCallbackFunction ) \

```

Comment:
I'm getting errs now with things like this macro being defined twice.  I will make some suggested edits then post back.

Reply:
I think I forgot to add the "WithCallback" to the second #define,  what I would like should be:
```
#if ( configUSE_SB_COMPLETED_CALLBACK == 1 )
    #define xStreamBufferCreate( xBufferSizeBytes, xTriggerLevelBytes ) \
        xStreamBufferGenericCreate( xBufferSizeBytes, xTriggerLevelBytes, pdFALSE, NULL)
    #define xStreamBufferCreateWithCallback( xBufferSizeBytes, xTriggerLevelBytes, pxStreamBufferCallbackFunction ) \
```